scout/adapter/mongo/user.py

```diff
@@ -82,14 +82,25 @@ def user(self, email=None, user_id=None):
             LOG.warning("No key provided to fetch user")
             return None
         query = {}
+        user_obj = None
         if user_id:
             LOG.info("Fetching user %s", user_id)
             query['_id'] = user_id
+            user_obj = self.user_collection.find_one(query)
```

Comment:
This query could be done also later in the code, maybe, once you have defined the query dictionary?

Reply:
Hm, I don't see a simpler way of doing that, with two queries in the else block? Feel free to edit! 😄